release.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
+#! /usr/bin/env python3
```

Comment:
Skipping this change, because of the extra space and because `release.py` is not intended to run in a venv.